mne/viz/utils.py

```diff
@@ -985,9 +989,9 @@ def _find_channel_idx(ch_name, params):
 
 def _draw_vert_line(xdata, params):
     """Draw vertical line."""
-    params['ax_vertline'].set_data(xdata, np.array(params['ax'].get_ylim()))
-    params['ax_hscroll_vertline'].set_data(xdata, np.array([0., 1.]))
-    params['vertline_t'].set_text('%0.3f' % xdata[0])
+    params['ax_vertline'].set_data(xdata[0], np.array(params['ax'].get_ylim()))
```

Comment:
Here `xdata` should now always be a tuple. It is strange that previously, in the first two lines of this function `xdata` was used, whereas in the third line `xdata[0]` was used. This can't be right.

Reply:
... actually maybe it's sufficient only to update using a singleton. In any case I assume using `xdata[0]` and using `vlines` should be sufficient going forward